### ISSUE TITLE
Spoiled metadata in fixed releases

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/backend#1244]
Some items(*) in GSD metadata in fixed CoL editions are changing to the latest view of metadata in CoL-Working Draft (id 3). That is wrong

(*) _at least **incorrect Alias & Abstract**_

**Comment by yroskov** [comment]
**Example from the 2023 Annual Checklist:**

AC23 was published with **3i Curculio** GSD. Authors of **3i Curculio** changed its Short Name to **Entiminae** a month after AC23 release (in July). But, in the list of AC23 sources (=COL23, id 9910), this GSD appears with a new name "Entiminae" (whereas should be "3i Curculio"):

https://www.checklistbank.org/dataset/9910/sourcemetrics

![image](https://github.com/CatalogueOfLife/backend/assets/28574252/19ebb6d3-a109-40dd-a5f5-e6de9851dcc5)

GSD page in AC23 shows correct Short Name and logo as "3i Curculio", but **incorrect Abstract** (!) which was changed a month after AC23 publication.
https://www.checklistbank.org/dataset/9910/source/1166

![image](https://github.com/CatalogueOfLife/backend/assets/28574252/909c55f9-ed5c-477e-a7f8-f51cac5c1165)

**Comment by yroskov** [comment]
Example from the CoL of August:

The same GSD Entiminae/former 3i Curculio (id 9910). Authors changed its **logo** after August release was started. But new logo appears in August release:

![image](https://github.com/CatalogueOfLife/backend/assets/28574252/92218363-cd08-4dfe-918e-999218cb45af)

Indeed, whatever logo I upload in CoL Working Draft in CLB, it will immediately appear in the fixed(!) edition on the portal.

For example, fake logo uploaded to CLB:
![image](https://github.com/CatalogueOfLife/backend/assets/28574252/55087fda-33a3-44ba-91e9-80c43b1c8568)

Logo immediately changed in August edition in the portal:
![image](https://github.com/CatalogueOfLife/backend/assets/28574252/e0f5eef9-3deb-487c-bfb9-810c77f9d3a0)

**Comment by yroskov** [comment]
So, Alias, Logo (what else from metadata) dynamically feeds from CoL Working Draft in CLB. 
**Supposed to be fixed monthly and annual editions editions may have incorrectly applied metadata from CoL Working Draft.**

**Comment by mdoering** [comment]
You are right about the logo, that was never archived and there is just a single one we keep per dataset.

**Comment by mdoering** [comment]
The metrics page uses this call to retrieve all datasets in one go:
https://api.checklistbank.org/dataset?limit=1000&contributesTo=9910&sortBy=alias

This does not query the archive, but takes the live dataset.
@thomasstjerne, can we change that to use this resource instead?
https://api.checklistbank.org/dataset/9910/source
